### PR TITLE
PullRequest

### PR DESCRIPTION
**Pull request by micksomo**

### CODE REVIEW

**Comment by Bobbey** [comment]
i dont like the code, can we please delete it and burn it with fire?

**Comment by micksomo** [comment]
Ich möchte gerne noch Console.WriteLine(); implementieren

File: Test123/Test123/Program.cs [hunk 1]
```diff
@@ -12,6 +12,15 @@ static void Main(string[] args)
         {
             Console.WriteLine("Hello World");
             Console.ReadKey(true);
+
+            //Looping Feature
+            for (int i = 0; i < 100; i++)
+            {
+                if (i%2 == 0)
+                {
```
Comment (Bobbey) [inline Test123/Test123/Program.cs:20]: that loop is not good

Comment (micksomo) [inline Test123/Test123/Program.cs:20]: WTF is this loop?

Comment (micksomo) [reply Test123/Test123/Program.cs:20]: I agree

Comment (Bobbey) [reply Test123/Test123/Program.cs:20]: i dont know, some drunk fella must have hacked into github and messed up our code!

**Pull request merged**